projects/netgrif-components/src/lib/data-fields/text-field/public-api.ts

```diff
@@ -0,0 +1,4 @@
+export * from './text-field.component';
+
+export * from './password-text-field/password-text-field.component';
+export * from './simple-text-field/simple-text-field.component';
```

Comment:
only these 3 components?

Reply:
This was only for testing purpose, I will remove this file.